006_plug_and_play_prior/toto.ipynb

```diff
@@ -0,0 +1,705 @@
+      "text/plain": [
+       "<Figure size 640x480 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mean_225 = np.reshape(samples.mean(), (N, N))\n",
+    "std_225 = np.reshape(samples.std(), (N, N))\n",
+    "\n",
+    "plt.figure()\n",
+    "IM_mean = samples.plot_mean()[0]\n",
+    "plt.colorbar(IM_mean)\n",
+    "plt.figure()\n",
+    "IM_std = samples.plot_std()[0]\n",
+    "plt.colorbar(IM_std)\n",
+    "plt.figure()\n",
+    "IM_ci = samples.plot_ci_width(0.95)[0]\n",
+    "plt.colorbar(IM_ci)\n",
+    "\n",
+    "plt.figure()\n",
+    "plt.imshow(im - mean_225, cmap = 'gray')\n",
+    "plt.title(r\"$im - Mean_{2.25}$\")\n",
+    "plt.colorbar()"
```

Comment:
Please explain what these images are, why they are shown and what we see/learn from them.

Reply:
We decided to show the mean, standard deviation and CI. I left one sentence there.